pkgs/os-specific/linux/rtl8188fu/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  name = "rtl8188fu-${kernel.version}-${version}";
+  version = "2022-06-11";
+
+  src = fetchFromGitHub ({
+    owner = "kelebek333";
+    repo = "rtl8188fu";
+  } // (if stdenv.targetPlatform.isAarch32 || stdenv.targetPlatform.isAarch64 then {
```

A comment explaining why ARM systems need its own rev/hash would be greatly appreciated
```suggestion
  } // (if stdenv.hostPlatform.isAarch then {
```